skimage/feature/texture.py

```diff
@@ -291,3 +293,130 @@ def local_binary_pattern(image, P, R, method='default'):
     image = np.ascontiguousarray(image, dtype=np.double)
     output = _local_binary_pattern(image, P, R, methods[method.lower()])
     return output
+
+def multiblock_local_binary_pattern(int_image, x, y, width, height):
```

Comment:
@vighneshbirodkar 
The time taken for `20x20` that has `2029` calls took `0.908968687057 sec`.
I didn't count to integral image calculation.
So, during training stage one epoch in case of `10000` training images will take `2` hours.
This is a lot, so we will for sure have to rewrite it.